OPERATOR_CONSOLE.md

```diff
@@ -0,0 +1,72 @@
+# Local Operator Console (LOC) API
+
+This document explains the Local Operator Console(LOC) from the API point of view.
+
+This document is similarly structured as PROFILE.md, because the topic is very similar.
+
+## Server endpoint
+
+EVE periodically sends a `CompoundEdgeDevConfigRequest` to the Local Operator Console.
+EVE can find the URL to the LOC in `loc_url` which is part of the
+`EdgeDevConfig` that EVE received from the controller.
+
+## Endpoints
+
+In the following `$UUID` is the uuid of the edge node contacting LOC.
+
+### Certs
+```http
+       GET /api/v2/edgedevice/certs
+```
+
+Returns the controller certificates.
```

Comment:
We should also document the payload for each endpoint. Name of the mime type (`application/x-proto-binary`), protobuf message name + link to its definition.